tests/cases/fourslash/server/postPasteImportFix_multiplePastes2.ts

```diff
@@ -0,0 +1,54 @@
+/// <reference path="../fourslash.ts" />
+
+// @Filename: /file1.ts
+//// import { aa, bb } from "./other";
+//// export const r = 10;
+//// export const s = 12;
+//// export const t = aa + bb + r + s;
+//// const u = 1;
+
+// @Filename: /target.ts
+//// /*a*/const a = 1;
+//// /*b*/const b = 2;
+//// const c = 3;
+//// const d = 4;
+
+// @Filename: /other.ts
+//// export const aa = 1;
+//// export const bb = 2;
+
+// @Filename: /tsconfig.json
+////{ "files": ["file1.ts", "target.ts", "other.ts"] }
+
+goTo.select("a", "b");
+format.setOption("insertSpaceAfterSemicolonInForStatements", true);
+verify.postPasteImportFix({
+    targetFile: "target.ts", 
+    pastes: [{
+        text: `export const t = aa + bb + r + s;
+const u = 1;\n`,
+        range: { pos: 13, end: 26 },
+    },
+    {
+        text: `export const t = aa + bb + r + s;
+const u = 1;\n`,
+        range: { pos: 39, end: 51 },
+    
+    }],
+    newFileContents: {
+        "/target.ts":
+`import { r, s } from "./file1";
+
+import { aa, bb } from "./other";
+
+const a = 1;
+export const t = aa + bb + r + s;
+const u = 1;
+const c = 3;
+export const t = aa + bb + r + s;
+const u = 1;
+`
+    }, 
+    originalFile: "file1.ts",
+    copyRange: { start: { line : 3, offset: 0}, end : { line: 4, offset: 12}},

```

Comment:
This is currently pasting same text at different cursor locations but it can be modified to paste different text at different locations as well. In that case, if the original file from which the code is copied is known, then  the `copyRange` will have to be an Array that points the correct copy range for every text. Is that something that we want to change?